roadmap/implementers-guide/src/node/dispute/votesdb.md

```diff
@@ -0,0 +1,139 @@
+# Votes DB
+
+Responsible module for tracking all kind of dispute votes by all validators
```

Comment:
I would amend this to say "all kinds of votes on candidates by all validators for a fixed number of sessions".